app/index.js

```diff
@@ -2380,6 +2371,157 @@ const saveWindowState = (statesOnly = false) => {
     }
 }
 
+/**
+ * Returns bookmarks file path.
+ */
+const getBookmarksFilePath = () => {
```

Comment:
can we leave this logic to be only in the bookmarks file? there is little to no need to write in in main, we can do the parsing here, but just send back the results to bookmarks and write it there so you can re-use this file logic.